src/UI/ledger/LedgerLoginContainer/AddressTable/AddressTable.tsx

```diff
@@ -69,6 +72,15 @@ export const AddressTable = ({
   const [accountsWithBalance, setAccountsWithBalance] = useState<
     Array<{ address: string; balance: string }>
   >([]);
+
+  const getFirstUnusedIndex = () => {
```

Comment:
Can you explain why this is needed? Since the `usedIndexes.includes(index)` should be enough to deal with primitive number types. Is this because, on page change, the new page has a reset index?